### PR TITLE
Demo walkthrough

### PR DESCRIPTION
**Pull request by kazbaig**
*Issue #, if available:*

*Description of changes:*
Based off #124.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/SimilarItemRecommendations.vue
Comment (alexchirayath) [inline src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/SimilarItemRecommendations.vue:16]: Lets remove the *and for tips on how to demonstrate this use case during customer meetings. * for now since this is not something that is providing info to the end customer

File: src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue
Comment (alexchirayath) [inline src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue:23]: There is a small change in the switch shopper image.
To be inclusive, the gender will  not be something that can be selected

File: src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue
Comment (alexchirayath) [inline src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue:30]: To be inclusive, the gender will  not be something that can be selected

File: src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/SimilarItemRecommendations.vue
Comment (kazbaig) [reply src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/SimilarItemRecommendations.vue:16]: Done

File: src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue
Comment (kazbaig) [reply src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue:23]: Will update once there is an implementation of the form that I can use for a screenshot.

File: src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue
Comment (kazbaig) [reply src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue:30]: Updated

**Comment by alexchirayath** [comment]
Since this branch consists of information related to features that have not been merged in (like SMS, Shopper select), please make sure to get James to review them and merged only after the other changes are merged.

File: src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue
Comment (alexchirayath) [reply src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue:23]: Sounds good. The rest of it looks good to me. Will approve once the image is updated

Comment (kazbaig) [reply src/web-ui/src/partials/AppModal/DemoWalkthrough/pages/Shoppers.vue:23]: Updated

**Review by alexchirayath — APPROVED** [review]
LGTM. Please get @james-jory  review once we are close to merge

**Comment by kazbaig** [comment]
Closing as this was merged as part of #142.

**Pull request closed without merging**
